bin/test

```diff
@@ -27,28 +46,33 @@ parser = ArgumentParser(
     epilog=epilog,
     formatter_class=ArgumentDefaultsHelpFormatter)
 parser.add_argument(
-    "-v", "--verbose", action="store_true", dest="verbose", default=False)
+    '--use-sympy-runner', action='store_true', dest='use_sympy_runner',
```

Comment:
If we're going to do this, at least add a short version of the flag (like `-s` or something) so that it doesn't require as much typing, especially if we plan to merge this in it's currently broken state.